lib/phi_attrs/railtie.rb

```diff
@@ -5,9 +5,10 @@
 
 module PhiAttrs
   class Railtie < Rails::Railtie
-    initializer 'phi_attrs.initialize' do
+    initializer 'phi_attrs.initialize' do |_app|
       ActiveSupport.on_load(:active_record) do
-        ActiveRecord::Base.send :extend, PhiAttrs
+        ActiveRecord::Base.send :extend, PhiAttrs::Model
+        ActionController::Base.send :include, PhiAttrs::Controller
```

Comment:
So this auto-extends the controller with the `PhiAttrs` stuff?

Reply:
Yes.